forge/build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id "com.github.johnrengelman.shadow" version "7.1.0"
+    id "com.github.johnrengelman.shadow" version "7.1.2"
```

Comment:
Maybe move the version into gradle.properties instead of duplicating it across mod loaders?